9-regular-expressions/06-regexp-boundary/1-find-time-hh-mm/task.md

```diff
@@ -1,9 +1,9 @@
-# Find the time
+# Encontre o horário
 
-The time has a format: `hours:minutes`. Both hours and minutes has two digits, like `09:00`.
+Um horário possui o formato `hours:minutes`. Ambas as horas e os minutos possuem dois dígitos, como `09:00`.
 
-Make a regexp to find time in the string: `subject:Breakfast at 09:00 in the room 123:456.`
+Componha uma regex que encontra o horário na string: `subject:Café da manhã as 09:00 no quarto 123:456.`
```

Comment:
I would undo the abbreviation in the word `regex` and just translate it as `expressão regular` because it sounds more clear during the reading.

Reply:
Makes sense! I'll merge it in later today